src/main/webapp/views/SearchPage.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <v-app>
+    <navbar></navbar>
+    <search-bar @searchresult="updateSearchResult"></search-bar>
+    <mock-search @searchresult="updateSearchResult"></mock-search>
+    <note-grid v-slot="{ filters }">
+      <note-grid-collection :note-data="searchResult"
+                            header="Most Favorited"
+                            :filters="filters"
+                            :compare-func="descFavorites">
+      </note-grid-collection>
+      <note-grid-collection :note-data="searchResult"
+                            header="Recently Posted"
+                            :filters="filters"
+                            :compare-func="descDate">
+      </note-grid-collection>
+      <!-- 
+        The following note-grid-collection sample is a rudimentary way of displaying notes 
+        that have gained the most score recently (1 week). In practice, the backend will need 
+        a way of counting the # of favorites and downloads a share has had in the past X days.
+      -->
+      <!--
+      <note-grid-collection :note-data="searchResult"
+                            header="Trending Notes"
+                            :filters="filters"
+                            :compare-func="descFavorites"
+                            :max-age="7 * 24 * 60 * 60 * 1000">
+      </note-grid-collection>
+      -->
```

Comment:
This is interesting, but I'm not 100% sure how we'd go about implementing a servlet to get "trending" notes on the backend with the existing schema. I feel like we'd need a `date_favorited` field on the favorite_notes table and to track downloads, we'd need a new downloaded_notes table that tracks a `note_id`, `user_id`, and `date_downloaded`. Having these tables also opens the possibility in the future for a note recommendations feature that sees all the labels of the notes a user has downloaded and favorited and shows them notes that have the same or similar labels.

Reply:
^ That's a cool idea for a new feature.